tests/examples_methods_syntax/multiple_interactions.py

```diff
@@ -76,13 +76,14 @@
 ).properties(title="Radio Button Highlighting")
 
 # Boolean selection for format changes
-input_checkbox = alt.binding_checkbox(name="Big Budget Films")
+input_checkbox = alt.binding_checkbox(name="Big Budget Films ")
 checkbox_selection = alt.param(bind=input_checkbox)
 
-size_checkbox_condition = alt.condition(checkbox_selection,
-                                        alt.SizeValue(25),
-                                        alt.Size('Hundred_Million_Production:Q')
-                                       )
+size_checkbox_condition = alt.condition(
+    checkbox_selection,
+    alt.Size('Hundred_Million_Production:Q'),
+    alt.SizeValue(25)
```

Comment:
I switched the order so that checking the box shows the big budget films instead of removes them